windows/vtpm_cold_add_remove/bitlocker_enable_test.yml

```diff
@@ -0,0 +1,109 @@
+        win_srv_bitlocker: "{{ win_powershell_cmd_output.stdout_lines[0].strip() }}"
+
+    - name: "BitLocker is already installed"
+      ansible.builtin.debug: var=win_srv_bitlocker
+      when: win_srv_bitlocker in ['Installed', 'Enabled']
+
+    - name: "BitLocker is not installed"
+      when: win_srv_bitlocker not in ['Installed', 'Enabled']     
+      block:
+        - name: "Install BitLocker"
+          include_tasks: ../utils/win_execute_cmd.yml
+          vars:
+            win_powershell_cmd: "Install-WindowsFeature BitLocker"
+        - name: "Restart guest OS after BitLocker install"
+          include_tasks: ../utils/win_shutdown_restart.yml
+          vars:
+            set_win_power_state: "restart"
+
+- name: "Enable BitLocker on 'C:' drive"
+  include_tasks: ../utils/win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "Enable-BitLocker -MountPoint 'C:' -UsedSpaceOnly -TpmProtector -SkipHardwareTest"
+  when: not win_bde_auto_enable
+
+- name: "Retry to get 'C:' drive encryption state"
+  ansible.windows.win_shell: "manage-bde -status 'C:'"
+  register: get_encrypt_result
+  ignore_errors: true
+  delegate_to: "{{ vm_guest_ip }}"
+  ignore_unreachable: true
+  retries: 60
+  delay: 10
+  until:
+    - get_encrypt_result.stdout is defined
+    - "'Used Space Only Encrypted' in get_encrypt_result.stdout"
+    - "'100.0%' in get_encrypt_result.stdout"
+
+- name: "Check 'C:' drive encryption state"
+  ansible.builtin.assert:
+    that:
+      - get_encrypt_result.failed is defined
+      - not get_encrypt_result.failed
+    fail_msg: "Not get 'Used Space Only Encrypted' and/or '100.0%' percentage encrypted result after 600 seconds."
+
+- name: "Get 'C:' drive encryption key protector"
+  include_tasks: ../utils/win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "manage-bde -protectors -get 'C:'"
+    win_execute_cmd_ignore_error: true
+
+- name: "Set fact of the result of getting key protector"
+  ansible.builtin.set_fact:
+    get_key_protector_result: "{{ win_powershell_cmd_output }}"
+
+# For Automatic Device Encryption enabled device, C: drive is encrypted automatically,
+# while key protector is None
+- name: "Add TPM protector"
+  when:
+    - win_bde_auto_enable
+    - "'TPM' not in get_key_protector_result.stdout | default('')"
+  block:
+    - name: "Add TPM protector for C: drive encryption"
+      include_tasks: ../utils/win_execute_cmd.yml
+      vars:
+        win_powershell_cmd: "manage-bde -protectors -add 'C:' -tpm"
+    - name: "Restart guest OS"
+      include_tasks: ../utils/win_shutdown_restart.yml
+      vars:
+        set_win_power_state: "restart"
+    - name: "Get 'C:' drive encryption key protector again"
+      include_tasks: ../utils/win_execute_cmd.yml
+      vars:
+        win_powershell_cmd: "manage-bde -protectors -get 'C:'"
+
+- name: "Check 'C:' drive key protector is TPM"
+  ansible.builtin.assert:
+    that:
+      - "'TPM' in win_powershell_cmd_output.stdout"
```

Comment:
Need to check `win_powershell_cmd_output.stdout is defined` firstly or add a default value like above `get_key_protector_result`.